Update install instructions (#74)
astropy/astropy-workshop: 00-Install_and_Setup/README.md

# Getting Started with Python and Anaconda

Please come to the workshop with a laptop already configured as described below.
If you have any problems with any of these steps, please

> [open an issue](https://github.com/astropy/astropy-workshop/issues).

Reviewer: maybe
```suggestion
[ask your question in the issue tracker](https://github.com/astropy/astropy-workshop/issues/new?assignees=&labels=workshop-question&template=question-from-workshop-participant.md&title=%5BQuestion%5D+Summarize+your+question+here).
```
? Unless we want them to see the issues list first...

## 1. Clone This Repository

Download the workshop folder using
[git](https://help.github.com/articles/set-up-git/):

    % git clone https://github.com/astropy/astropy-workshop.git

If you don't have git installed, you can download the ZIP file by pressing the
green *Clone or download* button at
https://github.com/astropy/astropy-workshop and selecting *Download ZIP*.
However, this option is not recommended because it impedes the ability to
update your copy of the repository if updates are made.

## 2. Install Miniconda (if needed)

*Miniconda is a free minimal installer for conda. It is a small, bootstrap

> version of Anaconda that includes only conda, Python, the packages they depend

Reviewer: (note somewhere in here that anaconda can be used if already installed)

Author: Is this already handled by your suggestion at a8ecb5a3108fb194f01a3c438f974ec1e9943b17 ?

Reviewer 2: I think it's covered.

on, and a small number of other useful packages, including pip, zlib and a few

> others.*

Reviewer: ```suggestion
others. Note, though, that if you have either miniconda or the full Anaconda 
already installed, you can skip to the next step.*
```

Check if Miniconda is already installed.

    % conda info

If Miniconda is not already installed, follow these instructions for your
operating system: https://docs.conda.io/en/latest/miniconda.html

On Windows, you might also need
[additional compilers](https://github.com/conda/conda-build/wiki/Windows-Compilers).

## 3. Create a conda environment for the workshop

> *Miniconda includes an environment manager called conda. You can create,

Reviewer: Maybe should include a definition of what an "environment" is?  Something like:
```suggestion
*Miniconda includes an environment manager called conda. Environments
allow you to have multiple sets of Python packages installed at the same 
time, making reproducibility and upgrades easier. You can create,
```
